capstone_project/src/main/java/com/google/sps/data/School.java

```diff
@@ -43,7 +39,28 @@ public double getLongitude() {
     return longitude;
   }
 
-  public ArrayList<String> getArticles() {
-    return articles;
+  @Override
+  public int hashCode() {
+    return (int) name.hashCode();
```

Comment:
I think that hashCode returns an int by default so you don't need to type cast it